### PR TITLE
feat: add possessions list

### PR DESCRIPTION
**Pull request by paddatrapper**
Captures the current state of #7

### CODE REVIEW

**Comment by jooola** [comment]
I think we can also list entries that have missing information, so we have a idea of what is missing.

Maybe we can merge this and add extra data when we  have it ?

File: possessions.md [hunk 1]
```diff
@@ -6,19 +6,29 @@ submitted there and approved for payment.
...
+| [OpenCollective](https://opencollective.com/libretime) | :check_mark: | @paddatrapper @robbt @jooola |
 
 ## Communication
 
-- [Twitter (@LibreTimeOSS)](https://twitter.com/LibreTimeOSS) - @hairmare has
-  the credentials to appoint admins and @gusaus is an admin
+| Service                                                             | Teams | Access                                                                  |
+| ------------------------------------------------------------------- | :---: | ----------------------------------------------------------------------- |
+| [Twitter @LibreTimeOSS](https://twitter.com/LibreTimeOSS)           |  :x:  | @hairmare has the credentials to appoint admins and @gusaus is an admin |
+| [YouTube](https://www.youtube.com/channel/UC-MA9GzkTb8th1YlDmF0NfA) |  :x:  | @Robbt                                                                  |
```
Comment (paddatrapper) [inline possessions.md]: I have admin for YouTube too

Comment (paddatrapper) [reply possessions.md]: It is a team account

File: possessions.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+| Service                                                |     Teams      | Access                       |
+| ------------------------------------------------------ | :------------: | ---------------------------- |
+| [OpenCollective](https://opencollective.com/libretime) | :green_circle: | @paddatrapper @robbt @jooola |
+
+## Communication
+
+| Service                                                             |     Teams      | Access               |
+| ------------------------------------------------------------------- | :------------: | -------------------- |
+| [Discourse](https://discourse.libretime.org/)                       |   :question:   | @robbt               |
+| [Mattermost](https://chat.libretime.org/)                           | :green_circle: | @paddatrapper        |
```
Comment (paddatrapper) [inline possessions.md]: There are other admins, but I host the instance and maintain the version. Other config is possible through the system console, admins are @paddatrapper, @gusaus, @hairmare, @jooola (now), @Robbt

**Pull request merged**